src/server/common/helpers/validate-service-is-not-a-test-suite.js

```diff
@@ -3,7 +3,7 @@ import Boom from '@hapi/boom'
 export function validateServiceIsNotATestSuite(request, h) {
   const service = request.app.service
 
-  if (service === null || service?.isTestSuite) {
+  if (service == null || service?.isTestSuite) {
```

Comment:
Sonar should have picked up on this. Should be `===`

Reply:
I had to change it to get the tests to pass. happy to discuss/demo

Reply:
I suspect `null` is a string then? Sounds like a bug?